File: 1264-maximum-number-of-words-you-can-type/maximum-number-of-words-you-can-type.java
class Solution {
    public int canBeTypedWords(String text, String brokenLetters) {
        int count = 0;
        String[] words = text.split(" ");
        for (String word : words) {
            for (char letter : word.toCharArray()) {
                if (brokenLetters.indexOf(letter) != -1) {
                    count++;
                    break;
                }
            }
        }
        return words.length - count;
    }
}

File: 1264-maximum-number-of-words-you-can-type/maximum-number-of-words-you-can-type.py
class Solution:
    def canBeTypedWords(self, text: str, brokenLetters: str) -> int:
        brokenLettersList = []
        splitText = text.split()
        count = 0
        for i in brokenLetters:
            brokenLettersList.append(i)
        for word in splitText:
            isBroken = False
            for b in brokenLettersList:
                if b in word:
                    isBroken = True
                    break
            if not isBroken:
                count += 1
        return count
